db/migrate/20220521183248_create_testimonials.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTestimonials < ActiveRecord::Migration[6.1]
+  def change
+    create_table :testimonials do |t|
+      t.string :name
```

Comment:
Es buena práctica en RoR que las restricciones estén presentes tanto en el modelo con el la base de datos:
```suggestion
      t.string :name, null: false
```